configure

```diff
@@ -2663,6 +2666,13 @@ else
   enable_pvode_openmp=no
 fi
 
+# Check whether --openmp-schedule was given.
+if test "${with_openmp_schedule+set}" = set; then :
+  enableval=$with_openmp_schedule;
```

Comment:
Do we want to validate that the user has provided a valid option or wait for the compilation to fail?

I think there's some way to provide a default without having to do this sort of check, but I can't remember. @ZedThree may be able to comment.